config.yaml

```diff
@@ -79,6 +79,7 @@ teams:
   maintainers:
   - tkuhrt
   members:
+  - jovfer
```

Comment:
```suggestion
  - jovfer
  members:
```
Shall we put this back the way that it was?

Reply:
no preference from my side, so up to you and @ryjones